lib/bootstrap.dart

```diff
@@ -96,21 +96,27 @@ class AppWrapper extends StatefulWidget {
 }
 
 class _AppWrapperState extends State<AppWrapper> with WidgetsBindingObserver {
```

Comment:
Remove `with WidgetsBindingObserver`

Reply:
I can't because we should listen didChangeLocales().

Reply:
Oh, ok. I thought it was used only for lifecycle